pkg/splunk/enterprise/types.go

```diff
@@ -58,6 +61,8 @@ func (instanceType InstanceType) ToRole() string {
 		role = "splunk_deployer"
 	case SplunkLicenseMaster:
 		role = "splunk_license_master"
+	case SplunkMonitoringConsole:
+		role = "splunk_monitor"
```

Comment:
How about naming it as "splunk_monitoring_conosole" to be in line with others?

Reply:
This is as per the naming convention in "ansible playbooks". Splunk monitoring console has "splunk_monitor" role